ert3/engine/_run.py

```diff
@@ -9,6 +9,7 @@ def _prepare_experiment(
     workspace_root: pathlib.Path,
     experiment_name: str,
     ensemble: ert3.config.EnsembleConfig,
+    ensemble_size: Optional[int],
```

Comment:
Does not seem like we ever pass `None` here, so I think we can drop the `Optional`.

Reply:
In type theory terms, `Optional[T]` means that the type can be either `T` or `None`. It doesn't mean that it necessarily needs to have a default value of `None`. Ie, `Optional[T]` can still be a required parameter.